contexts/credentials/v2

```diff
@@ -123,9 +123,9 @@
       }
     },
 
-    "VerifiableCredentialSchema2023": {
+    "JsonSchemaCredential2023": {
```

Comment:
```suggestion
    "JsonSchemaCredential": {
```

Are you keeping the date or removing it?

Reply:
removed